lib/lambda_ethereum_consensus/beacon/beacon_node.ex

```diff
@@ -55,8 +55,10 @@ defmodule LambdaEthereumConsensus.Beacon.BeaconNode do
       {LambdaEthereumConsensus.Beacon.BeaconChain, {anchor_state, time}},
       {LambdaEthereumConsensus.ForkChoice.Store, {anchor_state, anchor_block, time}},
       {LambdaEthereumConsensus.Beacon.PendingBlocks, []},
-      {LambdaEthereumConsensus.Beacon.SyncBlocks, []},
+      {LambdaEthereumConsensus.Libp2pPort, []},
```

Comment:
now `Libp2pPort` depends on BeaconChain to fetch the appropriate fork digest.

Reply:
Doesn't `PendingBlocks` make Libp2p requests? Shouldn't it be started later?

Reply:
Done